README.md

```diff
@@ -80,29 +88,15 @@ Once you have an external IP, you can proceed to visit that in your browser. The
 request any resources (memory/CPU/GPU), and then proceed to perform single node training.
 
 Note that when running on Google Kubernetes Engine, the public IP address will be exposed to the internet and is an 
-unsecured endpoint. For a production deployment, refer to the [detailed documentation](jupyterhub/README.md) on 
-how to set up SSL and authentication for your Hub. 
+unsecured endpoint. For a production deployment, refer to the [documentation](jupyterhub/README.md). 
 
 ### Training
 
-Please refer to the README in the [tensorflow/k8s](https://github.com/tensorflow/k8s) repository for more information on
-using the TfJob controller to run TensorFlow jobs on K8s.
+The TFJob controller takes a YAML specification for a master, parameter servers, and workers to help run [distributed tensorflow](https://www.tensorflow.org/deploy/distributed). The quick start deploys a TFJob controller and installs a new `tensorflow.org/v1alpha1` API type. 
+You can create new Tensorflow Training deployments by submitting a specification to the aforementioned API. For examples, look under the [tf-controller-examples/](https://github.com/google/kubeflow/tree/master/tf-controller-examples) directory.
 
-### Serve Model
-
-TODO(owensk)
-
-## Components
-
-### JupyterHub
+Refer to the README in the [tensorflow/k8s](https://github.com/tensorflow/k8s) repository for more information on using the TfJob controller to run TensorFlow jobs on K8s.
```

Comment:
I'm not fond of sending people to another repo to read up on how to use. Can we keep the instructions inline?

Reply:
The instructions are a bit complex - @jlewi PTAL

Reply:
Added some sample config but retained pointer to more detailed docs. @aronchick PTAL.